tests/core/mem-cgroup-disabled/task.yaml

```diff
@@ -0,0 +1,119 @@
+summary: Check basic functionality when memory cgroup is disabled
+
+# only run on UC20+ to use the convenient cmdline.extra file
+systems: [ubuntu-core-2*]
+
+environment:
+  SVC_UNIT: /etc/systemd/system/snap.test-snapd-simple-service.test-snapd-simple-service.service
+
+prepare: |
+  if ! os.query is-pc-amd64; then
```

Comment:
you could use not isntead of !

Reply:
same for execute

Reply:
I'll do these in a followup to be clear

Reply:
see https://github.com/snapcore/snapd/pull/11369